crates/protocol/src/utils.rs

```diff
@@ -8,22 +7,134 @@ where
+    fn test_flz_compress_len(#[case] input: &[u8], #[case] expected: u32) {
+        assert_eq!(flz_compress_len(input), expected);
     }
 
     #[test]
-    fn test_bytes_deposit() {
-        assert!(starts_with_2718_deposit(&Bytes::from_static(&[0x7E])));
-        assert!(!starts_with_2718_deposit(&Bytes::from_static(&[])));
-        assert!(!starts_with_2718_deposit(&Bytes::from_static(&[0x7F])));
+    fn test_flz_compress_len_no_repeats() {
+        let mut input = Vec::new();
+        let mut len = 0;
+
+        for i in 0..256 {
+            input.push(i as u8);
+            let prev_len = len;
+            len = flz_compress_len(&input);
+            assert!(len > prev_len);
+        }
```

Comment:
It seems challenging to port this test over https://github.com/bluealloy/revm/blob/ca5b2fe15e0b9920ede13f1c67918b93bfdc12c2/crates/optimism/src/fast_lz.rs#L154
since it reads the EVM state. 

Wondering if there's any suggestions for this

Reply:
We can add `revm` as a `dev-dependency` to use it for this test.